### ISSUE TITLE
Code for "hyp_mlr"

### ISSUE THREAD

**Issue by JinhwiPark** [MinaGhadimiAtigh/HyperbolicImageSegmentation#6]
Hi, thanks for the interesting paper.

I think the following part is the code for hyperbolic logit (paper equation(7)), and it seems that there is no $\lambda_{p_{y}}^{c}$ in the code:
https://github.com/MinaGhadimiAtigh/HyperbolicImageSegmentation/blob/622a17342911b3edb2a2845519963ed30f8ba755/hesp/util/layers.py#L115
Could you check this?

**Comment by seanzhuh** [comment]
Hi, I've encountered the same issue, and another problem I found out was that the input to tf.asinh does not divide w_y also

**Comment by jhindel** [comment]
I agree with @seanzhuh and @JinhwiPark. lambda is always set to 2 and w_y seems to be missing in the denominator. Is there any reason for it?
